scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/Logging.kt

```diff
@@ -11,7 +11,7 @@ package io.seldon.dataflow
 
 import io.klogging.Level
 import io.klogging.config.loggingConfiguration
-import io.klogging.rendering.RENDER_ANSI
+import io.klogging.rendering.RENDER_ISO8601
```

Comment:
**Review note**
Moving to non-color logger as it produces noisy escape sequences when viewing the logs outside k9s